src/content/en/2021/ecommerce.md

```diff
@@ -691,6 +693,8 @@ In our research, we found that less than 1% of websites had third-party accessib
 
 Further information on such tools can be found in the [Accessibility](./accessibility) chapter.
 
+{# TODO (Authors) - can we link specifically into the captcha section once the link is available? #}
```

Comment:
@bobbyshaw

Reply:
Added this as working on that chapter. Resolving.